### DISCUSSION TITLE
Use bun instead of npm

### DISCUSSION THREAD

**Discussion by onx2** [inokawa/virtua#490] [Ideas]
A quick test between the two on a fresh install and bun in ~29 seconds faster. Other than perf reasons it probably isn't a big deal but I've used it in some personal projects and have enjoyed it without any issue (as a package manager). WDYT?

**Bun**
<img width="500" alt="image" src="https://github.com/user-attachments/assets/31943780-a5ed-415a-a031-b9484a2fdb25">

**NPM**
<img width="465" alt="image" src="https://github.com/user-attachments/assets/e706b9c3-e175-4167-aee0-51d02f0176c1">

**Comment by inokawa** [comment]
https://github.com/inokawa/virtua/pull/491#issuecomment-2279401564
